java/src/main/java/com/logicalclocks/base/FeatureGroupBase.java

```diff
@@ -1,35 +1,29 @@
 /*
- * Copyright (c) 2020 Logical Clocks AB
+ *  Copyright (c) 2020-2023. Hopsworks AB
```

Comment:
make abstract and remove `@NoArgsConstructor` (probably we discussed it already, just making sure)

Reply:
Yes we did but I can't make this class as it is used in multiple places to be initialised

Reply:
done :)